src/components/myPage/ProfilePicChanger.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState } from 'react';
+
+const ProfilePicChanger: FC<{
```

Comment:
I guess these are the codes that you are going to clean?

Reply:
nice! those pictures are really hilarious lol it updates the profile pics! really cool

Reply:
indeed yes.